XYZDrivers/pom.xml

```diff
@@ -47,8 +60,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
Why did you change the version of java? I think we were meant to use 1.7 and I'm not sure if the team have 1.8 installed on their machines

Reply:
I have 1.8 installed on my machine but can't speak for anyone else. I think we should stick with 1.7 though as that's probably what the assignment spec asks for.

Reply:
I did it to get lambdas, but since you don't have it I'll convert away from them. Also a good call just in case the demo computers don't have Java 8.